modules/exploits/unix/webapp/wp_cm_download_manager_exec.rb

```diff
@@ -0,0 +1,95 @@
+##
+# This module requires Metasploit: http://metasploit.com/download
+# Current source: https://github.com/rapid7/metasploit-framework
+##
+
+require 'msf/core'
+
+class Metasploit3 < Msf::Exploit::Remote
+  Rank = NormalRanking
+
+  include Msf::Exploit::Remote::HttpClient
+
+  def initialize(info = {})
+    super(update_info(info,
+      'Name'           => 'Wordpress CM Download Manager Plugin Code Injection',
+      'Description'    => %q{
+        This module exploits a vulnerability in the alterSearchQuery() function
+        defined in /wp-content/plugins/cm-download-manager/lib/controllers
+        /CmdownloadController.php for Wordpress CM Download Manager plugin
+        versions 2.0.0 and earlier. User input passed through 'CMDsearch'
+        GET parameter isn't properly sanitized before being used in a call
+        to preg_match_all().  This can be exploited to inject and execute
+        arbitrary code leveraging the PHP's complex curly syntax.
+      },
+      'Author'         => 'Nixawk', # originally reported by the (Phi Ngoc Le)
+      'License'        => MSF_LICENSE,
+      'References'     =>
+        [
+          ['EDB', '35324']
+        ],
```

Comment:
WPVULNDB Reference to add:
https://wpvulndb.com/vulnerabilities/7679
(other references are also linked on wpvulndb if you want to add them)